packages/catlog/src/dbl/model.rs

```diff
@@ -210,14 +185,9 @@ where
         }
     }
 
-    /// Discrete double theory that the model is of.
-    pub fn theory(&self) -> &Arc<DiscreteDblTheory<Cat>> {
-        &self.theory
-    }
-
-    /// Graph of object and morphism generators for the model.
-    pub fn generating_graph(&self) -> &impl FinGraph<V = Id, E = Id> {
-        self.category.generators()
+    /// get the Arc for the theory of this model
```

Comment:
Not important right now, but I just used `Arc` instead of say `archery` to keep things simple. I gather the overhead is minimal compared to the `Rc`. But it would be nice to have some common guidelines. I noticed you used `archery` in the directories implementation.

Reply:
I think archery would be overkill here. This is a impl for the struct, not the trait, so we can be specific here.